README.md

```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
 
 The projects use [npm semantic versioning](https://docs.npmjs.com/about-semantic-versioning)
 
+## Project boards
+
+Between the 5th of June and 17th of July, there is a sprint board used for organising the backlog for each two weeks sprint cycle.
+
+- [BBC News Labs - Digital Paper Edit - Sprint Board](https://github.com/orgs/bbc/projects/33) is a github project board [with linked repository](https://help.github.com/en/articles/linking-a-repository-to-a-project-board) where issues are added from the relevant repository listed in architecture section.
+
+The project is divided into [concurrent milestones as described here](https://github.com/bbc/digital-paper-edit-client/milestones)
+
+And there are [github project board for each milestone](https://github.com/bbc/digital-paper-edit-client/projects)
+
+- [1 - Transcript - Editing](https://github.com/bbc/digital-paper-edit-client/projects/2)
+- [2 - Transcript - Annotating](https://github.com/bbc/digital-paper-edit-client/projects/3)
+- [3 - Paper-editing](https://github.com/bbc/digital-paper-edit-client/projects/4)
+
+With UX being an overarching thing that cuts across these different parts
+- [UX](https://github.com/bbc/digital-paper-edit-client/projects/1)
+
+
+## UX
```

Comment:
Yeah I agree with your comment that I'm not sure this section belongs here. I'd move to Confluence :)

Reply:
ok, moved it to separate section in docs